template/App.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react'
 import {Provider} from 'react-redux'
-import {PersistGate} from 'redux-persist/integration/react'
 import {store, persistor} from './src/store/store'
+import {PersistGate} from 'redux-persist/lib/integration/react'
```

Comment:
Why are we changing this? The example in the latest docs is still `import {PersistGate} from 'redux-persist/integration/react'`

Reply:
Outdated